winui/Kanban/cards.md

```diff
@@ -0,0 +1,161 @@
+---
+layout: post
+title: Cards in WinUI Kanban control | Syncfusion
+description: This section describes about cards in Syncfusion WinUI Kanban (SfKanban) control, its elements and more.
+platform: winui
+control: SfKanban
+documentation: ug
+---
+
+# Cards in WinUI Kanban Control
```

Comment:
Change to - `Cards in WinUI Kanban (SfKanban) control`

Reply:
Changed.